oss/sdks/ts/vector/commands/query.mdx

```diff
@@ -33,6 +33,15 @@ The query method is designed to retrieve the most similar vectors from the index
       The metadata of the vector. This is used to make it easier to identify the
       vector on queries.
     </ResponseField>
+    <ResponseField name="includeVectors" type="boolean">
```

Comment:
`includeVectors` is duplicated?

Reply:
Yeah, somehow it got duplicated. I'll fix it.